psydac/polar/dense.py

```diff
@@ -277,6 +277,10 @@ def update_ghost_regions( self, *, direction=None ):
     def ghost_regions_in_sync(self):
         return True
 
+    # ...
+    @ghost_regions_in_sync.setter
+    def ghost_regions_in_sync( self, value ):
+        self._sync = True
```

Comment:
Are you not ignoring the `value` here? You should check that it is a `bool` and then set `self._sync` accordingly.

Reply:
I thought that the ghost regions are always True in this class

Reply:
Yes, but then it makes little sense to have a setter... I see, you call this from a `BlockVector` and you want it to have no effect.

Many methods in this class only contain a `pass` statement. I suggest you do the same here, and do not set `self._sync` at all.

Reply:
okay

Reply:
done in [9ed13cb](https://github.com/pyccel/psydac/pull/232/commits/9ed13cb7ef3cc0b2cf5ef6284c62b95ffd2874f1)